etc/env/example.env

```diff
@@ -2,7 +2,7 @@
 # - - -
 
 # site options
-CANDIG_MODULES=weavescope minio drs-server htsget-server chord-metadata candig-server candig-data-portal #wes-server jupyter igv-js traefik portainer consul logging monitoring datasets cnv-service rnaget cancogen-dashboard federation-service
+CANDIG_MODULES=weavescope minio drs-server htsget-server chord-metadata candig-server federation-service candig-data-portal #wes-server jupyter igv-js traefik portainer consul logging monitoring datasets cnv-service rnaget cancogen-dashboard
```

Comment:
Please leave federation-service as manual deploy for now. I am writing doc for how to deploy in separate branch and PR